contracts/proxy/diamond/base/DiamondBaseStorage.sol

```diff
@@ -7,12 +7,12 @@ pragma solidity ^0.8.20;
  */
 library DiamondBaseStorage {
     struct Layout {
-        // function selector => (facet address, selector slot position)
-        mapping(bytes4 => bytes32) facets;
+        // function selector => (facet address, selector slug position)
+        mapping(bytes4 => bytes32) selectorInfo;
         // total number of selectors registered
         uint16 selectorCount;
-        // array of selector slots with 8 selectors per slot
-        mapping(uint256 => bytes32) selectorSlots;
+        // array of 32-byte slugs with 8 selectors each
```

Comment:
Could this be?

```
// array of Bytes32 slugs. Each Bytes32 contains 8, 4byte selectors.
```

Reply:
I think we use `array` because the slug is indexed by `uint256`. Were using the mapping like an array.

Reply:
Yes.  I believe it's a mapping because an array's `length` would be updated in storage on each push or pop.  Might be more efficient to use an array anyway, but use assembly to avoid the length update.  Can look into this later.